classes/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand
```

Comment:
- [ ] [OPTIONAL] `rand` function generates a random number between 0 and 1, usually a long decimal number. Please specify a range to generate a random number from, that way it'll make it easier to use it in the future. 😎️
**Example**
`rand(1..1000)`